arabic/GetYourselfOutOfHereForAMovie/PickAMovieAndGoAndTakeARest

```diff
@@ -0,0 +1 @@
+	pick from here http://www.imdb.com/chart/top?ref_=nv_mv_250_6
```

Comment:
yeah , merge

On Sun, May 15, 2016 at 11:55 PM, Casey Collaborator <
notifications@github.com> wrote:

> In arabic/GetYourselfOutOfHereForAMovie/PickAMovieAndGoAndTakeARest
> <https://github.com/udacity/create-your-own-adventure/pull/4512#discussion_r63298583>
> :
>
> > @@ -0,0 +1 @@
> > +	pick from here http://www.imdb.com/chart/top?ref_=nv_mv_250_6
>
> Please make sure you name all of your files with the extension .md.
> Otherwise, the files will not display correctly on GitHub.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/4512/files/3d1da03a9b84f21dbd4fedf3786f2eb3bd16c7ab#r63298583>
>



-- 
Seif Mostafa